### ISSUE TITLE
[FIX] Remove attribute `label` from tag `label` to fix way of reading by Screen Readers

### ISSUE THREAD

**Issue by sonya0504** [infermedica/component-library#381]
## Describe the bug
Tag `label` should have attribute `label` and Screen Reader read this label 3 times.

## To Reproduce
1. Turn on Screen Reader
2. In storybook open UiMultiSelectChoices
3. You heart label 3x

## Expected behavior
Tag `label` doesn't have attribute `label` ([documentation](https://www.w3schools.com/tags/ref_attributes.asp))
You can hear `label` only once

## Screenshots
<img width="1399" alt="image" src="https://github.com/infermedica/component-library/assets/37744371/4f5734b5-598e-455b-b5d4-ff1c800852f5">
